src/app/components/Searchbar/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { View, Image, TextInput } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { useSelector, useDispatch } from 'react-redux';
+
+import removeIcon from '@assets/ic_remove.png';
+import searchPlaceholderIcon from '@assets/ic_search_placeholder.png';
+import { AppState } from '@interfaces/appState';
+import bookActions from '@redux/books/actions';
+
+import styles from './styles';
+
+function Searchbar() {
+  const dispatch = useDispatch();
+  const input = useSelector((state: AppState) => state.book.querySearch);
+  const setInput = (textInput: string) => dispatch(bookActions.setValue('querySearch', textInput));
```

Comment:
Let's call this `setSearch` or something like that. I think this is cool, but we don't pass the target of an action on out projects. Hardcode `'querySearch'` into the action